Cargo.toml

```diff
@@ -76,6 +76,7 @@ recursive = "0.1"
 regex = "1.9"
 reqwest = { version = "0.12", default-features = false }
 ryu = "1.0.13"
+scopeguard = "1.2.0"
```

Comment:
We want to reduce dependencies already. I don't think its worth a whole new dependency.

Reply:
OK

Reply:
This is already a transitive dependency, for what it's worth, since it's a very popular library.